modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -248,7 +248,6 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         int type;
         std::vector<size_t> kernel_size, strides;
         std::vector<size_t> pads_begin, pads_end;
-        CV_DEPRECATED_EXTERNAL Size kernel, stride, pad;
         CV_DEPRECATED_EXTERNAL int pad_l, pad_t, pad_r, pad_b;
```

Comment:
Will it be complicated to remove deprecated paddings as well?

Reply:
> Will it be complicated to remove deprecated paddings as well?

Done.